reviews/incubation-falco.md

```diff
@@ -0,0 +1,180 @@
+# Falco Incubation Stage Review Proposal
+
+The goal of the Falco project is to provide cloud native runtime security with a focus on Kubernetes intrusion & anomaly detection. Falco achieves this through the use of eBPF, Linux kernel probes, and the Kubernetes audit logging API to create an event stream of low level system and orchestrator events. This includes network connection info, file opens/writes, process information, and Kubernetes API calls. Falco also polls metadata (labels, resource names, container info, etc) from the Kubernetes API server and the underlying container runtime. 
+
+The Falco engine applies rule sets (or policies) to this event stream to identify potentially malicious behavior, or to simply log activity on a Kubernetes cluster for later auditing or review. The Kubernetes metadata can be incorporated into the rule conditions to allow rules to apply to only particular resources in a Kubernetes cluster. Falco alerts can be sent to a wide range of destinations including logging arggreation services, pub/sub services, serverles functions, and more. This allows Falco users to integrate Falco alerts into their existing incident response and Ops/SRE workflows.
+
+Falco entered the CNCF Sandbox on October 11th, 2018. Please see the project's [Sandbox proposal](https://github.com/cncf/toc/blob/master/proposals/falco.adoc). 
+
+## Project Progress since Sandbox
+
+In short, the Falco project has seen terrific growth and project participation since entering the CNCF Sandbox. 
+
+* All key metrics around contributions and committers have seen significant growth.
+* Downloads have seen significant growth.
+* The project has seen an increase in integrations, as well as Falco being incorporated into other products (Sumo Logic, Altran).
+* The community has seen increased participation and activity.
+* Public end users are documented in the projects' [ADOPTERS.md](https://github.com/falcosecurity/falco/blob/dev/ADOPTERS.md). Two end users are speaking at Kubecon NA 2019.
+* The project completed a security audit and fixed 14 security bugs.
```

Comment:
is the security audit published? I can't find it easily

Reply:
I should revise that. We've fixed 13/14 bugs and will publish the audit when the last one is fixed. We are aiming to get this in next couple of releases.